include/dali/c_api.h

```diff
@@ -325,23 +325,36 @@ DLL_PUBLIC unsigned daliGetNumOutput(daliPipelineHandle *pipe_handle);
 DLL_PUBLIC device_type_t daliGetOutputDevice(daliPipelineHandle *pipe_handle, int id);
 
 /**
- * @brief Copy the output tensor stored
- * at position `n` in the pipeline.
+ * @brief Copy the output batch stored at position `output_idx` in the pipeline.
  * @remarks If the pipeline output is TensorList then it needs to be dense
  * @param pipe_handle Pointer to pipeline handle
- * @param device Device of the supplied memory.
  * @param dst Pointer to the destination buffer where the data will be copied
- * @param output_id index of the pipeline output
+ * @param output_idx index of the pipeline output
  * @param dst_type Device type associated with the destination buffer (0 - CPU, 1 - GPU)
- * @param stream CUDA stream to use when copying the data onto GPU. Remember to synchronize on the
- *               provided stream.
+ * @param stream CUDA stream to use when copying the data to/from the GPU.
+ *               Remember to synchronize on the provided stream.
```

Comment:
What does it mean to "synchronize on a stream"?

Reply:
`cudaStreamSynchronize(stream)` I assume ;)

Reply:
I'll remove this remark (which was copied from the previous implementation). It's nothing related to what our API does. The API launches a copy on a stream. Whether you want to synchronize after or not, is up to your application